### PR TITLE
add fps-transfers endpoint and define uk-urls for apm and legal-notice

### PR DESCRIPTION
**Pull request by klausmeyer**
:gb: :tada:

### CODE REVIEW

**Comment by igorkosta** [comment]
@klausmeyer looks good! Could you maybe add some information about the `fps` related transactions with `transaction_type_details`.

Here is an example:

```
{
      "id": "66666",
      "account_id": "66666666",
      "transaction_type": "fps_payout",
      "subject": "Crystal Meth",
      "amount": -6666,
      "booking_code": "750",
      "booking_date": "2016-03-09T00:00:00+00:00",
      "value_date": "2016-03-09T00:00:00+00:00",
      "return_transaction_id": "",
      "created_at": "2016-03-09T14:12:39Z",
      "updated_at": "2016-03-09T16:15:09Z",
      "currency": "GBP",
      "transaction_type_details": {
        "fps_transfer_id": "66666",
        "remote_name": "Walther White",
        "remote_account": "66666666",
        "remote_sort_code": "404761"
      }
    }
```

**Comment by klausmeyer** [comment]
@igorkosta sure. I've pushed an update.

**Comment by igorkosta** [comment]
@klausmeyer one last thing: http://fidor.github.io/api-docs-uk/#transaction-type-details there is a table with different `transaction types` and a short description. After that we're good to go :)

**Comment by klausmeyer** [comment]
@igorkosta I thought I've already added two new lines to this table:
https://github.com/fidor/api-docs-uk/pull/1/files#diff-9dab1d9b26f6bbbc376602c6acfda1e8R401

**Comment by igorkosta** [comment]
@klausmeyer you're absolutely right - my bad - sorry!

**Comment by klausmeyer** [comment]
Not a problem :bow:

File: source/includes/_money_transfer.md [hunk 1]
```diff
@@ -103,6 +103,81 @@ expired | Transfer expired and the money was returned
...
+created_at | Creation date-time, never changes | String (date-time) ISO 8601 Date-Time |
+updated_at | Last update date-time | String (date-time) ISO 8601 Date-Time |
+
+
+### HTTP Request
+`GET https://api.fidor.de/fps_transfers`  <sub>index</sub>
+
+`GET https://api.fidor.de/fps_transfers/{id}`  <sub>self</sub>
+
+`POST https://api.fidor.de/fps_transfers`  <sub>create</sub>
```
Comment (weiss100) [inline source/includes/_money_transfer.md]: These examples should be changed to api.fidorbank.uk as well. But that's true for all the examples.

Comment (klausmeyer) [inline source/includes/_money_transfer.md]: You are right. I'll submit a second PR for that.

**Pull request merged**